### DISCUSSION TITLE
Should the upgrade guide link on the documentation site work?

### DISCUSSION THREAD

**Discussion by ilmarivacklinsc** [aws-powertools/powertools-lambda-typescript#2085] [Q&A] [answered]
Right now the link goes to: https://docs.powertools.aws.dev/lambda/typescript/upgrade

When I click on it, I get this:

```xml
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>index.html</Key>
<RequestId>P7YYZF0ZDDXTNEM9</RequestId>
<HostId>
D8HRWdnkaLV/8edXVYKt2jmQFaYL8KP9vghINwxAazuSe7r/KYqKnOMga6035S9MNcViZ2VHqfo=
</HostId>
</Error>
```

I feel there could be a better UX for this if the upgrade guide isn't available yet.

**Comment by dreamorosi** [comment, marked as answer]
Hi, thanks for reporting the issue.

The upgrade guide is available ([here](https://docs.powertools.aws.dev/lambda/typescript/latest/upgrade/)) but the link in the announcement banner is indeed broken.

I have opened an issue (#2090) and will fix it right away. In the meantime you can use the other link in the left-side menu bar.

**Reply by ilmarivacklinsc** [reply]
Thanks!

**Reply by dreamorosi** [reply]
We have just fixed & [deployed](https://github.com/aws-powertools/powertools-lambda-typescript/actions/runs/7962317083) the docs again.

If you don't see the new version yet, try clearing the browser cache or use private browsing.

Thank you again for reporting the issue!
